### ISSUE TITLE
I got kubernetes on top of Windows Docker desktop running but how to test it?

### ISSUE THREAD

**Issue by PavelSosin** [docker/for-win#3423]
- [X ] I have tried with the latest version of my channel (Stable or Edge)

My landscape:
Windows 10 fall enterprise edition
Hyper-V driver
Docker Desktop for Windows
  
Finally, I got Kubernetes on top of Docker Desktop for Windows running according the Whale windows but how to test it?  I have no Kubernetes dashboard deployed
### Expected behavior
Kubernetes dashboard deployed and available and available via https:8080/dashboard
### Actual behavior
https://localhost:8080/dashboard - localhost refused to connect.

### Information

  - Windows Version:
  - Docker for Windows Version:

### Steps to reproduce the behavior

  1. ...
  2. ...

**Comment by PavelSosin** [comment]
I'm almost their! After execution of

kubectl proxy
and deploying dashboard as 
kubectl create https://raw.githubusercontent.com/kubernetes/dashboard/master/aio/deploy/recommended/kubernetes-dashboard.yaml
I got it at https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/

But now I need authentication via kubctl config. Where is my kubctl config file ??


-

**Comment by PavelSosin** [comment]
I'm almost their! After execution of

kubectl proxy
and deploying dashboard as
kubectl create https://raw.githubusercontent.com/kubernetes/dashboard/master/aio/deploy/recommended/kubernetes-dashboard.yaml
I got it at https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/

But now I need authentication via kubctl config. Where is my kubctl config file ??

**Comment by PavelSosin** [comment]
I found my kubectl konfig file but its content is obviously wrong - it is configured for minikube instance installed previously on my computer. How to configure entry for my kubernetes. I expected that it is generated automatically.

**Comment by v-schulz** [comment]
I have the same issue:
when activating kubernetes for windows the config file in $HOME/.kube/config is not changed. The UI states that Kubernetes is up and running and the containers are created (docker ps). However without the changes to the config file connecting to the cluster is not possible. I tried some config file I found in some tutorial but then I am still missing the key and certificate.
I also tried reinstalling Docker, reset to factory defaults, delete the previous .kube folder with minikube configuration.
Docker Version: 18.09.2
